pyproject.toml

```diff
@@ -167,6 +167,9 @@ responses = ">=0.22.0,<0.23.0"
 [tool.poetry.group.docs]
 optional = true
 
+[tool.flakehell]
+extended_default_ignore=[]
```

Comment:
Right. I've updated my setup later and I wanted to remove this too. I have `flake8 v6.0.0` and it doesn't even need flake8-pyproject (it seems to be included automatically) to work with pyproject.toml.

Reply:
Are you sure it's not coming from some other package somehow? flake8/issues/234 makes it seem like there's quite a bit of opposition to adopting pyproject.toml, so I'd be surprised if flake8 did this itself. Maybe one of our dependencies depends on something like flake8-pyproject, pyproject-flake8, flakehell, flakeheaven, etc. (Those are the ones I'm aware of... :facepalm: )

Reply:
Right. I don't know to be honest. I saw that it's listed with `flake8 --version` and thought it might be integrated but flake8 might just list all its installed plugins.

Reply:
You're right. It's not installed by flake8 v6.0.0. I don't know what installs it.